About/About/faq.md

```diff
@@ -2,13 +2,13 @@
 # FAQ
 
 ## How do open source community health metrics help me understand whether an open source project is healthy?
-CHAOSS provides standardized metrics that help generate important information about open source projects you are interested in. You can use CHAOSS metrics to see changes in your project that might indicate changes in project health. You can also use CHAOSS metrics to compare your project to similiar projects. At the CHAOSS Project, we do not offer an opinion about the health of any project but provide objective measures that can help you make informed decisions.
+CHAOSS provides standardized metrics that help generate important information about open source projects you are interested in. You can use CHAOSS metrics to see changes in your project that might indicate changes in project health. You can also use CHAOSS metrics to compare your project to similar projects. At the CHAOSS Project, we do not offer an opinion about the health of any project but provide objective measures that can help you make informed decisions.
 
 ## Will you create a project health index?
 No, open source projects work very differently and we believe that a single index will not do this diversity justice. Our approach, rather, is to provide a collection of standardized metrics that can inform your personal opinion about project health.
 
 ## What do you mean with 'implementation-agnostic metrics'?
 Our goal is to define metrics independent of a specific implementation. At the CHAOSS project, we provide the tools by which you can make more informed decisions regarding the open source projects that are important to you -- in your specific context. 
 
-## What are the IP or commitments that contributing members are making to make sure that the derived IP is patent free and distributed under an open source license.
-Our goal is to make our innovations available to everyone to use for free. We honor open source licenses of upstream projects. We license our own work under the GPLv2+ and MIT open source licenses. We only accept patent free contributions or where patent licenses are available for free.
+## What are the IPs or commitments that contributing members are making to make sure that the derived IP is patent free and distributed under an open source license?
+Our goal is to make our innovations available to everyone to use for free. We honour open source licenses of upstream projects. We license our own work under the GPLv2+ and MIT open source licenses. We only accept patent free contributions or where patent licenses are available for free.
```

Comment:
US vs. British terminology: honor vs. honour. Your suggestion is fine.

Reply:
ok